dataplane/redbeams/redbeams.go

```diff
@@ -227,28 +228,32 @@ func DeleteDatabaseServer(c *cli.Context) {
 	output.Write(serverListHeader, row)
 }
 
-var dbListHeader = []string{"Name", "Description", "Crn", "EnvironmentCrn", "DatabaseVendor", "ConnectionURL"}
+var dbListHeader = []string{"Name", "Type", "Description", "Crn", "EnvironmentCrn", "DatabaseVendor", "ConnectionURL", "ResourceStatus", "CreationDate"}
 
 type dbDetails struct {
 	Name           string `json:"Name" yaml:"Name"`
+	Type           string `json:"Type" yaml:"Type"`
 	Description    string `json:"Description" yaml:"Description"`
 	CRN            string `json:"CRN" yaml:"CRN"`
 	EnvironmentCrn string `json:"EnvironmentCrn" yaml:"EnvironmentCrn"`
 	DatabaseEngine string `json:"DatabaseEngine" yaml:"DatabaseEngine"`
 	ConnectionURL  string `json:"ConnectionURL" yaml:"ConnectionURL"`
+	ResourceStatus string `json:"ResourceStatus" yaml:"ResourceStatus"`
 	CreationDate   int64  `json:"CreationDate" yaml:"CreationDate"`
 }
 
 func (db *dbDetails) DataAsStringArray() []string {
-	return []string{db.Name, db.Description, db.CRN, db.EnvironmentCrn, db.DatabaseEngine, db.ConnectionURL, string(db.CreationDate)}
+	return []string{db.Name, db.Type, db.Description, db.CRN, db.EnvironmentCrn, db.DatabaseEngine, db.ConnectionURL, db.ResourceStatus, strconv.FormatInt(db.CreationDate, 10)}
 }
 
 func NewDetailsFromDbResponse(r *model.DatabaseV4Response) *dbDetails {
 	details := &dbDetails{
```

Comment:
I think the following fields could also go into dbDetails: Description, DatabaseEngine. I might have missed the reason why these were not included, sorry for that :)

Reply:
They do go in there. If you look a little further down, they are added to the details object if they are not nil.